src/bento/siteWideConfig.js

```diff
@@ -1,5 +1,8 @@
 export default {
   // Suggested for replaceEmptyValueWith: 'N/A' or '-' or ''
   replaceEmptyValueWith: '',
+  // Enable authenication
   enableAuthentication: true,
+  // List for options for authentication empty array defaults to google
+  authEndPoint: ['google'], // authEndPoint: []
```

Comment:
Don't we need URL for "authEndPoint" property ?

Reply:
we don't need it if nothing is given its defaults to google

Reply:
all google login defaults to single place